components/Notifications.tsx

```diff
@@ -66,7 +74,23 @@ export default function Notifications() {
                               ? 'Transaction succeeded'
                               : 'Transaction pending'}
                             <span className="inline-block ml-2 text-indigo-500 text-xs">
-                              {description}
+                              {functionName}
+                            </span>
+                            <span className="inline-block ml-2 text-indigo-500 text-xs">
+                              {crossChain && (
```

Comment:
nit: since this is a boolean, usually prefer `isCrossChain` here and elsewhere. feel free to ignore.